### PR TITLE
Fix verify and test-go CI jobs

### PR DESCRIPTION
**Pull request by fejta**
Will migrate gotest-dockerized.sh over to test-infra in the next PR

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/1067)

### CODE REVIEW

**Comment by spxtr** [comment]
https://github.com/kubernetes/kubernetes/issues/36796

File: jobs/ci-kubernetes-verify-release-1.5.sh [hunk 1]
```diff
@@ -25,7 +25,8 @@ export KUBE_VERIFY_GIT_BRANCH='release-1.5'
 export KUBE_TEST_SCRIPT="./hack/jenkins/verify-dockerized.sh"
 
 ### Runner
-timeout -k 15m 80m "${KUBE_TEST_SCRIPT}" && rc=$? || rc=$?
+readonly runner="./hack/jenkins/gotest-dockerized.sh"
+timeout -k 15m 80m "${runner}" && rc=$? || rc=$?
```
Comment (krousey) [inline jobs/ci-kubernetes-verify-release-1.5.sh]: I'm not sure I understand. Does `KUBE_TEST_SCRIPT` affect what happens in `gotest-dockerized.sh`?

Comment (krousey) [inline jobs/ci-kubernetes-verify-release-1.5.sh]: answered my own question

**Pull request merged**